core/vapp-python/.project-creation/run.py

```diff
@@ -32,9 +37,26 @@ def verbose_copy(src, dst) -> object:
 
 def clone_sdk(sdk_temp_dir: str) -> None:
     with open(f"{os.path.dirname(__file__)}/config.json") as f:
-        repo_url = json.load(f)["sdkUri"]
+        config = json.load(f)
```

Comment:
For better maintainability and separation of concerns, you want to read the config elsewhere and pass sdkUri and sdkVersion as string parameters to the `clone_sdk`  function.